docs/server/settings.rst

```diff
@@ -250,6 +250,147 @@ Configuration settings for applications used by Pootle.
   - ``label`` specifying the text that will be displayed next to the mark.
 
 
+.. setting:: POOTLE_REPORTS_INVOICES_PHANTOMJS_BIN
+
+``POOTLE_REPORTS_INVOICES_PHANTOMJS_BIN``
+  Default: ``None`` (unset)
+
+  .. versionadded:: 2.7.3
+
+  Absolute path to the `PhantomJS binary <http://phantomjs.org/>`_. This is
+  needed in order to optionally generate PDF invoices out of user activity.
+
+
+.. setting:: POOTLE_REPORTS_INVOICES_DIRECTORY
+
+``POOTLE_REPORTS_INVOICES_DIRECTORY``
+  Default: ``working_path('invoices')``
+
+  .. versionadded:: 2.7.3
+
+  Base directory where the invoices will be created. Invoices will be generated
+  in subdirectories for each year and month combination, i.e.
+  ``$POOTLE_REPORTS_INVOICES_DIRECTORY/<YYYY-MM>/``.
+
+
+.. setting:: POOTLE_REPORTS_INVOICES_COMPANY
+
+``POOTLE_REPORTS_INVOICES_COMPANY``
+  Default: ``'ACME'``
+
+  .. versionadded:: 2.7.3
+
+  Name of your company as displayed in the invoice. Check the templates at
+  *templates/invoices/* for further details and customizations.
+
+
+.. setting:: POOTLE_REPORTS_INVOICES_DEPARTMENT
+
+``POOTLE_REPORTS_INVOICES_DEPARTMENT``
+  Default: ``'ACME Department'``
+
+  .. versionadded:: 2.7.3
+
+  Name of your company deparment as displayed in the invoices. Check the
+  templates at *templates/invoices/* for further details and customizations.
+
+
+.. setting:: POOTLE_REPORTS_INVOICES_RECIPIENTS
+
+``POOTLE_REPORTS_INVOICES_RECIPIENTS``
+  Default: ``{}`` (empty dictionary)
+
+  .. versionadded:: 2.7.3
+
+  The list of usernames for whom invoices will be generated and send to.
+
+  The setting holds a dictionary where the keys are actual usernames of the
+  running Pootle instance and the values are dictionaries of key-value pairs
+  which will hold the user-specific configuration used to construct individual
+  invoices.
+
+  The user configuration accepts the following keys:
+
+  ``name`` (required)
+    Full name as displayed in the invoice. The official "full" person name can
+    be different from the public *shortened* version available in the Pootle
+    profile, hence this name is used instead as it's dealing with financial
+    documents.
+
+  ``wire_info`` (required)
+    Wire information which includes payment details.
```

Comment:
I think we can mention that this can be a multi-line string, and that such a multi-line string will properly be rendered as a multi-line one in the invoice. Also, to allow better formatting of the configuration files, we will trim leading and trailing spaces on each line of this multi-line value.